mcpserver/args.go

```diff
@@ -0,0 +1,40 @@
+package mcpserver
+
+import "github.com/mark3labs/mcp-go/mcp"
+
+var (
+	explainationArgument = mcp.WithString("explanation",
```

Comment:
🐛 **Bug (High):** Typo in variable name: 'explainationArgument' should be 'explanationArgument' for consistency and correctness

```suggestion
	explainationArgument = mcp.WithString("explanation",
```